dotcom-rendering/src/client/startup.ts

```diff
@@ -1,17 +1,25 @@
-import { log, startPerformanceMeasure } from '@guardian/libs';
+import { log } from '@guardian/libs';
+import { startPerformanceMeasure } from '../lib/startPerformanceMeasure';
 
 const measure = (name: string, task: () => Promise<void>): void => {
-	const { endPerformanceMeasure } = startPerformanceMeasure('dotcom', name);
+	try {
+		const { endPerformanceMeasure } = startPerformanceMeasure(
+			'dotcom',
+			name,
+		);
 
-	task()
-		.then(() => {
-			const duration = endPerformanceMeasure();
-			log('dotcom', `🥾 Booted ${name} in ${duration}ms`);
-		})
-		.catch(() => {
-			const duration = endPerformanceMeasure();
-			log('dotcom', `🤒 Failed to boot ${name} in ${duration}ms`);
-		});
+		task()
+			.then(() => {
+				const duration = endPerformanceMeasure();
+				log('dotcom', `🥾 Booted ${name} in ${duration}ms`);
+			})
+			.catch(() => {
+				const duration = endPerformanceMeasure();
+				log('dotcom', `🤒 Failed to boot ${name} in ${duration}ms`);
+			});
+	} catch (error) {
+		// performance APIs are unsupported or broken…
+	}
```

Comment:
This is quite defensive and probably unnecessary… but we can revert this PR once we’ve fixed upstream. As noted by @jamesgorrie